src/firefly/js/ui/UploadTableChooser.js

```diff
@@ -268,28 +265,30 @@ export function showUploadTableChooser(setUploadInfo,groupKey= 'table-chooser',d
 
 const TapUploadPanel= ({setUploadInfo,groupKey= 'table-chooser',defaultColsEnabledObj}) => {
     return (
-    <FieldGroup groupKey={groupKey}>
-        <Stack style={{ resize: 'both', overflow: 'hidden', zIndex: 1, pt:1, minWidth: 600, minHeight: 500}}>
-            <FieldGroupTabs initialState={{value: 'upload'}} fieldKey='upload-type-tabs' groupKey={groupKey}
-                            sx={{width: 1,  flex: '1 1 auto'}}>
-                <Tab name='Upload Tables' id='upload' sx={{fontSize:'larger'}}>
-                    <FileUploadDropdown {...{
-                        sx:{height:1,
-                            '.ff-FileUploadViewPanel-file':{ml:3},
-                            '.ff-FileUploadViewPanel-acceptList':{ml:3},
-                        },
-                        acceptOneItem:true, acceptList:[TABLES], keepState:true, groupKey:groupKey+'-fileUpload',
-                        onCancel:() => dispatchHideDialog(dialogId),
-                        onSubmit:(request) => uploadSubmit(request,setUploadInfo,defaultColsEnabledObj),
-                    }}/>
-                </Tab>
-                <Tab name='Loaded Tables' id='tableLoad' sx={{fontSize:'larger'}}>
-                        <LoadedTables {...{
-                            style:{height: '100%', width:'100%'}, keepState: true, groupKey:groupKey+'-tableLoad',
-                            onCancel:() => dispatchHideDialog(dialogId),
-                            onSubmit:(request) => existingTableSubmit(request,setUploadInfo,defaultColsEnabledObj)
-                        }}/>
-                </Tab>
-            </FieldGroupTabs>
-        </Stack>
-    </FieldGroup> );};
+        <Stack height='35rem' sx={{resize:'both', overflowY:'auto',minHeight:'35rem', minWidth:'40rem'}}>
+            <FieldGroup groupKey={groupKey} sx={{ flexGrow: 1 }}>
```

Comment:
@robyww can you take a look at the `overflyY:auto` I added here? 
Reason: after you upload a table, if you make the panel bigger, and then make it smaller again, you lose the `Load` and `Cancel` buttons (probably needs more cleanup later to just fix this problem - but for now adding `overflowY:auto` ensures user will still have access to those buttons by scrolling).